packages/clerk-js/src/ui/components/UserProfile/ActiveDevicesSection.tsx

```diff
@@ -61,6 +61,7 @@ const DeviceAccordion = ({ session }: { session: SessionWithActivitiesResource }
           sx={t => ({
             alignItems: 'flex-start',
             padding: `${t.space.$2} ${t.space.$4}`,
+            marginLeft: `-${t.space.$4}`,
```

Comment:
Why do we use negative margins?

Reply:
We need to position the button to the left to align text with the items in the list